exporter/datadogexporter/go.mod

```
@@ -24,6 +24,7 @@ require (
 	gopkg.in/zorkian/go-datadog-api.v2 v2.30.0
 	k8s.io/apimachinery v0.24.3
 	k8s.io/client-go v0.24.3
+    github.com/DataDog/gohai v0.0.0-20220718130825-1776f9beb9cc 
```

```suggestion
	github.com/DataDog/gohai v0.0.0-20220718130825-1776f9beb9cc 
```
Some tabs vs spaces thing happened here.